source/manual/2nd-line.html.md

```diff
@@ -58,29 +58,65 @@ See the 2nd line Trello board section for more information.
 
 NB: Technical 2nd Line should investigate these alerts when there is downtime; you do not necessarily have to fix them.
 
+### Grafana
+
+We use Grafana dashboards to monitor the health of our applications and service across our environments ([Integration](https://grafana.blue.integration.govuk.digital/?orgId=1), [Staging](https://grafana.blue.staging.govuk.digital/?orgId=1), [Production](https://grafana.blue.production.govuk.digital/). Some useful dashboards include:
+
+- Second line, which includes data from our Origin health and Edge health dashboards
+- Sidekiq
+- Application deployment dashboards
```

Comment:
Out of interest, why did you pick these particular dashboards? Are these copied from somewhere?

Reply:
This was written by Deb I think - not sure as to the rationale.

Reply:
Fair enough. They seem sensible to go with.